docs/reference/commandline/volume_create.md

```diff
@@ -32,7 +32,7 @@ Multiple containers can use the same volume in the same time period. This is use
 Volume names must be unique among drivers.  This means you cannot use the same volume name with two different drivers.  If you attempt this `docker` returns an error:
 
 ```
-A volume named  %s  already exists with the %s driver. Choose a different volume name.
+A volume named  <conflict_name>  already exists with the <origin_driver> driver. Choose a different volume name.
```

Comment:
I think we can use the volume name that is used in the examples above here, e.g.

```Markdown
A volume named hello already exists with the some-other driver. Choose a different volume name.
```

(Looking at that, it would be nice if the actual error message would use `%q` to quote the names)